closed/autoconf/custom-hook.m4

```diff
@@ -184,6 +185,9 @@ AC_DEFUN_ONCE([OPENJ9_THIRD_PARTY_REQUIREMENTS],
 
     AC_MSG_NOTICE([Could not find freemarker.jar])
     AC_MSG_ERROR([Cannot continue])
+  else
+    AC_MSG_RESULT([yes])
+    AC_CHECK_FILE($with_freemarker_jar,, AC_MSG_ERROR([freemarker.jar not found in directory indicated]))
```

Comment:
We can be more helpful by echoing the path the user specified (which shouldn't be a directory):
```
freemarker jar not found at $with_freemarker_jar
```

Reply:
I have updated with that suggestion.